src/components/Footer.jsx

```diff
@@ -1,20 +1,175 @@
+  margin: 0 auto;
+
+  @media (min-width: 1200px) {
+    padding: 0rem 15.25rem;
+  }
+
+  .footer-content {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+
+    @media (min-width: 1200px) {
+      width: 653.16px;
+    }
+  }
+
+  .footer-links {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+
+    @media (min-width: 1200px) {
+      flex-direction: row;
+    }
+  }
+
+  .footer-socials-container {
+    display: flex;
+    align-items: center;
+    list-style-type: none;
+    padding: 0;
+  }
+
+  .donate-button-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 0.5rem;
+    padding: 0.5rem 1rem;
+    background-color: var(--white);
+    border: 2px solid #d90000;
+    box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 2.2px;
+  }
+
+  .internal-links {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    list-style-type: none;
+    gap: 2rem;
+  }
+
+  .external-links {
+    display: flex;
+    align-items: center;
+    list-style-type: none;
+    gap: 1rem;
+    padding: 0;
+  }
+  .social-img {
+    min-width: 1.5rem;
+  }
+
+  .footer-text {
+    text-align: center;
+    color: var(--light-grey);
+    font-weight: 400;
+
+    @media (min-width: 768px) {
+      text-align: center;
+    }
+
+    a {
+      color: var(--cfa-blue-dark);
+      text-decoration: none;
+    }
+  }
 `
 const Footer = () => {
   return (
     <StyledFooter>
-      <img src={logo} alt="Code for Chicago Logo Icon" />
-      <p>
-        <a href="https://codeforamerica.org/" target="_blank" rel="noreferrer">
-          Code for America Labs, Inc.
-        </a>{' '}
-        is a non-partisan, non-political 501(c)(3) charitable organization
-      </p>
+      <div className="footer-content">
+        <div className="footer-links">
+          <div className="donate-button-container">
+            <a
+              href="https://www.codeforamerica.org/donate-to-a-brigade?utm_campaign=Code%20for%20Chicago&utm_source=Brigade%20site"
+              target="blank"
+              rel="noopener noreferrer"
+              className="button donateButton"
+            >
+              Donate
+            </a>
+
+            <img src={longArrowAltRight} alt="" />
+          </div>
+          <ul className="internal-links">
+            <li activeClassName="active">
+              <Link to="/projects">Projects</Link>
+            </li>
+            <li activeClassName="active">
+              <Link to="/portfolio">Portfolio</Link>
+            </li>
+            <li activeClassName="active">
+              <Link to="/join">Join Us</Link>
+            </li>
+            <li activeClassName="active">
+              <Link to="/about">About</Link>
+            </li>
+          </ul>
+          <div className="footer-socials-container">
```

Comment:
Let's see if we can get rid of this div and combine the styles with the `ul.external-links` to simplify the markup.